class TaskNotFoundError(Exception):
    pass

class TaskNotOwnedByUserError(Exception):
    pass 